lib/Base/CMakeLists.txt

```diff
@@ -6,7 +6,11 @@ add_library(Base
 
 target_link_libraries(Base
                       PUBLIC
-                        Support)
+                        Support
+                        glog)
+
+target_include_directories(Base PUBLIC ${GLOW_THIRDPARTY_DIR}/glog)
```

Comment:
who is going to put it there (`${GLOW_THIRDPARTY_DIR}/glog`)?

Reply:
Ah I need to remove this. I originally was going to go this route but I realized depending on glog externally is maybe better since it's so readily available.